src/controllers/petsController.js

```diff
@@ -1,19 +1,32 @@
 const petsModel = require('../models/petsModel.js');
 
-const { getAllPets } = petsModel;
+const { getAllPets, postNewPet } = petsModel;
 
 const getPets = async (req, res, next) => {
-	await getAllPets()
-		.then((dbResponse) => {
-			res.send(dbResponse);
-		})
-		.catch((e) => {
-			console.log(e.message);
-			res.sendStatus(500);
-			next(e);
-		});
+  await getAllPets()
+    .then((dbResponse) => {
+      res.send(dbResponse);
+    })
+    .catch((e) => {
+      console.log(e.message);
+      res.sendStatus(500);
+      next(e);
+    });
+};
+
+const postPet = async (req, res, next) => {
+  await postNewPet(req.body.newPet)
+    .then((dbResponse) => {
+      res.send(dbResponse);
```

Comment:
Our controller needs to return the status along with the body. We also need to add error handling in the controllers. Per the Project Plan document on Confluence, the responses for this endpoint are 201 Created, 400 Bad Request, 401 Unauthorized, 406 Not Acceptable

We can probably leave 401 for now, and wait until all the user verification stuff is made

Reply:
- 201 Created - added
- 400 Bad Request - added
- 401 Unauthorized - I added an empty middleware 'requireAuth' in /utils/auth.js and used it for POST /pets. I think it would be cleaner if we could make use of the middleware in server.js for all routes that need it and we don't have to worry about the 401 error anymore in the controllers. You can update the middleware later with the user verification part. Let me know if you have other thoughts!
- 406 Not Acceptable - for POST requests I didn't use any accept headers so I'm not sure when to return this error. I don't think we need any accept header but let me know if you disagree.